code/game/machinery/firealarm.dm

```diff
@@ -32,6 +32,9 @@
 	light_range = 7
 	light_color = "#ff3232"
 
+	//Trick to get the glowing overlay visible from a distance
+	luminosity = 1
```

Comment:
This isn't needed since we already fixed it on bee

Reply:
addressed in 1389b29